samples/nrf9160/modem_shell/src/utils/mosh_print.c

```diff
@@ -84,15 +84,15 @@ void mosh_fprintf(enum mosh_print_level print_level, const char *fmt, ...)
 
 	k_mutex_lock(&mosh_print_buf_mutex, K_FOREVER);
 
+	va_start(args, fmt);
```

Comment:
Should there be a matching `va_end()` call before the `return` below?

Reply:
That'd make sense, I've added a common exit point.